toonz/sources/toonz/frameheadgadget.h

```diff
@@ -86,4 +87,18 @@ class FilmstripFrameHeadGadget : public FrameHeadGadget {
   bool eventFilter(QObject *obj, QEvent *event);
 };
 
+namespace ZeroThickToggleGui {
```

Comment:
@turtletooth 
I think the implementation is fine. But IMO these code should be moved from frameheadgadget.h/.cpp to sceneviewercontextmenu.h/.cpp as this feature will be used only in the viewer's context menu. The other part of frameheadgadget sources are for the Level Strip, so mixing them may cause confusion.